### PR TITLE
TASK: Add english translations

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: Configuration/NodeTypes.Lists.yaml [hunk 1]
```diff
@@ -29,15 +29,30 @@
...
-        label: 'Sort order'
+        label: 'i18n'
         reloadIfChanged: true
         inspector:
           group: 'list'
+          editor: 'TYPO3.Neos/Inspector/Editors/SelectBoxEditor'
+          editorOptions:
+            values:
+              'DESC':
+                label: 'Descending'
```
Comment (dimaip) [inline Configuration/NodeTypes.Lists.yaml]: Why is this label not translated?

**Comment by johannessteu** [comment]
Just missed them :)
Commit ist updated

**Comment by dimaip** [comment]
Terrific! Thanks a lot! I didn't get to polishing ListNode yet, thanks for doing it instead of me!

**Comment by johannessteu** [comment]
You're welcome!

**Pull request merged**